lib/httparty/connection_adapter.rb

```diff
@@ -126,6 +133,14 @@ def connection
         http.open_timeout = options[:open_timeout]
       end
 
+      if options[:write_timeout] && (options[:write_timeout].is_a?(Integer) || options[:write_timeout].is_a?(Float))
+        if RUBY_VERSION >= "2.6.0"
```

Comment:
There is a lot of duplication as well. Probably a would be better, to create a wrapper method like so _(just an idea, not sure about the naming)_
```ruby
for_versions_above('2.6', option: :write_timeout) do
  http.write_timeout = options[:write_timeout]
end
```